src/database/operations/load/loadProposal.ts

```diff
@@ -2,9 +2,13 @@ import { db } from '../../db';
 import { NotFoundError } from '../../../errors';
 import type { JsonResultSet, Proposal } from '../../../types';
 
-export const loadProposal = async (id: number): Promise<Proposal> => {
+export const loadProposal = async (
+	id: number,
+	createdBy: number = 0,
```

Comment:
I'm of two minds about it. First, why complicate it? But then I suppose it doesn't make sense to select a pile of joined stuff and later look at it on the application level to verify that it has the right user.

Reply:
Resolved by pre-flight auth select.